sound/soc/sof/sof-priv.h

```diff
@@ -445,6 +445,9 @@ struct snd_sof_dev {
 	wait_queue_head_t waitq;
 	int code_loading;
 
+	/* probes */
+	struct hdac_ext_stream *extractor;
```

Comment:
can you clarify what you need a stream for the extractor but not the injector?
I am guessing this has to do with N inject and 1 extract but I am reading what I want to read here.

Reply:
Precisely. One cannot start two extractions as only one is supported. There are many probe points available for it though. In injection case, you can have many of them but only 1 probe point per injection stream.

Guess this code is also "out" to align with no-HDA SOF dependent goal.